cob_navigation_slam/CMakeLists.txt

```diff
@@ -13,24 +13,16 @@ install(DIRECTORY launch config
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
 
-# rostest for robots
-# SET(robotlist
-# cob3-1
-# cob3-2
-# cob3-3
-# cob3-4
-# cob3-5
-# cob3-6
-# raw3-1
-# raw3-2
-# raw3-3
-# raw3-4
-# desire
-# )
-#
-# # loop through list of robots for testing
-# FOREACH(robot ${robotlist})
-#     MESSAGE("testing for robot: ${robot}")
-#     roslaunch_add_file_check(launch ROBOT=${robot})
-#     roslaunch_add_file_check(launch robot:=${robot})
-# ENDFOREACH(robot)
\ No newline at end of file
+#############
+## Testing ##
+#############
+
+# if(CATKIN_ENABLE_TESTING)
+#   find_package(cob_supported_robots REQUIRED)
+#   find_package(roslaunch REQUIRED)
+#   foreach(robot ${cob_supported_robots_ROBOTLIST})
+#     message("testing for robot: ${robot}")
+#     roslaunch_add_file_check(launch ROBOT=${robot})    # testing environment variables
+#     roslaunch_add_file_check(launch robot:=${robot})   # testing launch file arguments
+#   endforeach()
+# endif()
```

Comment:
fair enough :laughing: 

I'd say activate all three.

However, thinking about this, we should probably make sure to also remove the now unsupported robots according to your work in cob_robots. Also, I'm not sure if we have all environments fully set-up in cob_environments. But activating the tests will force us to do this :grin: